libpff/libpff_attachment.c

```diff
@@ -55,7 +55,7 @@ int libpff_attachment_get_type(
 	libpff_record_entry_t *record_entry   = NULL;
 	libpff_record_set_t *record_set       = NULL;
 	static char *function                 = "libpff_attachment_get_type";
-	uint32_t attachment_method            = 0;
+	uint32_t attachment_method            = LIBPFF_ATTACHMENT_TYPE_UNDEFINED;
```

Comment:
So LIBPFF_ATTACHMENT_TYPE_UNDEFINED is 0 and is not the same as attachment method. So I'll not adopt this change. Seeing that remaining change is a 1 liner I'll add to a batch of some other changes.